src/js/app.jsx

```diff
@@ -48,7 +49,8 @@ library.add(
     faAngleRight,
     faChevronDown,
     faChevronUp,
-    faSpinner
+    faSpinner,
+    faInfoCircle
```

Comment:
At some point (perhaps in this pr!?) I think we should replace the info icon with this icon for the IDV Tooltips! I defer to your judgment on whether to do that in the scope of this PR.